examples/tests/valid/if.w

```diff
@@ -0,0 +1,12 @@
+bring cloud;
+
+let x = 1;
+if x == 2 {
+  print("x is 2");
+}
+
+new cloud.Function(inflight (s: str): str => {
+  if x == 1 {
+    print("x is 1");
+  }
+}) as "func";
```

Comment:
Turn this into a real test:

```suggestion
bring cloud;

let x = 1;
let var ok = false;
if x == 2 {
  print("x is 2");
  ok = true;
}
assert(ok);

new cloud.Function(inflight (s: str): str => {
  let var ok = false;
  if x == 1 {
    print("x is 1");
    ok = true;
  }
  assert(ok);
}) as "test";
```

Reply:
I will change the bool to avoid the issue #1109